LeanplumTests/LeanplumTests.m

```diff
@@ -17,23 +21,65 @@ @interface LeanplumTests : XCTestCase
 @implementation LeanplumTests
 
 - (void)setUp {
-    // Put setup code here. This method is called before the invocation of each test method in the class.
+    [super setUp];
+    [LPTestHelper setup];
 }
 
 - (void)tearDown {
-    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+    [OHHTTPStubs removeAllStubs];
 }
 
-- (void)testExample {
-    // This is an example of a functional test case.
-    // Use XCTAssert and related functions to verify your tests produce the correct results.
+/**
+ * Tests whether setting user attributes and id works correctly.
+ */
+- (void) test_user_attributes
+{
+    [LPTestHelper setupStub:200 withFileName:@"simple_success_response.json"];
```

Comment:
did you add this json file? I renamed the the one i made to simple_post_success....

Reply:
yea, i will need to rebase my code with master to get your new file names.